pkg/util/fdchannel.go

```diff
@@ -59,7 +59,7 @@ func RecvMsg(via net.Conn) (int, []byte, error) {
 
 	klog.V(4).Info("calling recvmsg...")
 	buf := make([]byte, syscall.CmsgSpace(4))
-	b := make([]byte, 500)
+	b := make([]byte, 5000)
```

Comment:
How do we know 5000 will handle most of the mountOption cases? Is there a way to make this a resizable buffer like Vikas suggested in our eng sync on March 4?